crates/solidity/outputs/cargo/slang_solidity/src/resolver.rs

```diff
@@ -0,0 +1,85 @@
+use metaslang_bindings::PathResolver;
+
+pub struct SolidityPathResolver;
```

Comment:
I believe we won't have a "default" resolver, and the user will always need to provide one if they want to resolve imports. Otherwise, all imports will stay unresolved.

If so, WDYT about moving `SolidityPathResolver` to tests, instead of exporting it from the public API?

Reply:
Sounds good.